spanner/spanner_snippets/snippet_test.go

```diff
@@ -45,6 +45,10 @@ func TestSample(t *testing.T) {
 	defer adminClient.Close()
 	defer dataClient.Close()
 
+	// Issue a pessimistic delete of the database before starting, as it's possible that resources have
+	// not been fully cleaned up from prior runs.
+	t.Logf("pessimistic spanner delete result: %v", adminClient.DropDatabase(ctx, &adminpb.DropDatabaseRequest{Database: dbName}))
```

Comment:
Maybe we should check if the database exists first? No need to log an error if delete doesn't work because the database doesn't exist (and shouldn't).

Reply:
Good point.  Will augment to check beforehand.